### PR TITLE
Disassemble yellow sounds

### PR DESCRIPTION
**Pull request by PikalaxALT**
There's only one `dr` left in `audio.asm`, and it's most likely uninitialized bytes

### CODE REVIEW

File: audio/sfx/unknown_8058b.asm [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+	unknownsfx0x20 $2, $f7, $60, $06
+	unknownsfx0x20 $1, $e7, $40, $06
+	unknownsfx0x20 $1, $e7, $20, $06
+	unknownsfx0x20 $f, $d1, $00, $06
+	unknownsfx0x20 $4, $c7, $40, $07
+	unknownsfx0x20 $4, $a7, $30, $07
+	unknownsfx0x20 $f, $91, $20, $07
+	endchannel
+
+SFX_805ae_4_Ch5
```
Comment (dannye) [inline audio/sfx/unknown_8058b.asm]: Missing colon

File: audio/headers/sfxheaders4.asm [hunk 2]
```diff
@@ -1,5 +1,5 @@
 SFX_Headers_4:: ; 80000 (20:4000)
-	db $ff, $ff, $ff
+	dbw -1, -1 ; padding
```
Comment (dannye) [inline audio/headers/sfxheaders4.asm]: This is misleading and unnecessary.

File: audio/music/jessieandjames.asm [hunk 3]
```diff
@@ -0,0 +1,303 @@
...
+	rest 1
+	octave 3
+	A# 1
+	rest 1
+	octave 4
+	C# 1
+	rest 1
+	E_ 1
+	rest 1
+	loopchannel 0, .loop
```
Comment (dannye) [inline audio/music/jessieandjames.asm]: Put line endings at EOFs.

File: audio.asm [hunk 4]
```diff
@@ -733,7 +735,7 @@ INCLUDE "audio/music/surfingpikachu.asm"
 INCLUDE "audio/music/yellowunusedsong.asm"
 INCLUDE "audio/music/jessieandjames.asm"
 
-	dr $832b9,$84000
+INCBIN "audio/unknown_832b9.bin"
```
Comment (dannye) [inline audio.asm]: Just why

**Comment by dannye** [comment]
"uninitialized bytes"?
It's not like bytes can be uninitialized and then that means they get to be random. Their purpose may be outdated and unused, but they're almost certainly not garbage. In either case, it is premature to decide.

File: engine/overworld/doors.asm [hunk 5]
```diff
@@ -25,33 +25,21 @@ IsPlayerStandingOnDoorTile: ; 1a609 (6:6609)
...
-	dw LobbyDoorTileIDs
-	db MANSION
-	dw MansionDoorTileIDs
-	db LAB
-	dw LabDoorTileIDs
-	db FACILITY
-	dw FacilityDoorTileIDs
-	db PLATEAU
-	dw PlateauDoorTileIDs
+DoorTileIDPointers: ; 1a7a8 (6:67a8)
```
Comment (YamaArashi) [inline engine/overworld/doors.asm]: We want to keep pokered and pokeyellow in sync as much as possible, so if you make edits like this, could you do the same in pokered?

**Pull request merged**